app/models/repository_host/raw_upstream_data_converter.rb

```diff
@@ -52,7 +52,7 @@ def self.convert_from_gitlab_api(api_project)
       is_private: api_project.visibility != "public",
       keywords: api_project.topics,
       language: nil, # separate API endpoint that doesn't seem to be supported by the API gem we use,
-      license: api_project.license.key,
+      license: api_project.license&.key,
```

Comment:
Is this related to the status change....?

Reply:
It's not, but I did encounter it when playing with some Gitlab requests